apps/backend/src/referentiels/import-referentiel/import-referentiel.service.ts

```diff
@@ -387,6 +402,27 @@ export class ImportReferentielService extends BaseSpreadsheetImporterService {
             );
           }
         }
+
+        if (action.indicateurs) {
+          this.logger.log(
+            `Action ${actionId} is linked to indicateurs: ${action.indicateurs.join(
+              ','
+            )}`
+          );
+          action.indicateurs?.forEach((identifiant) => {
+            const existing_relations = indicateurIdentifiants.find(
```

Comment:
Un détail mais cette constante devrait être au singulier non ? Au début j'ai cru à une erreur (que tu avais mis un `find` plutôt qu'un `filter`)
```suggestion
            const existingRelation = indicateurIdentifiants.find(
```

Reply:
Oui je ne sais pas pourquoi. L'IA m'a piégé peut être ?  Je vais être plus vigilant. Merci en tout cas.